### ISSUE TITLE
deprecating legacy plot API

### ISSUE THREAD

**Issue by edublancas** [ploomber/jupysql#513]
once https://github.com/ploomber/jupysql/pull/508 is merged, we should start showing deprecation warnings:

https://github.com/ploomber/jupysql/blob/8ce3334182733145dbea6ece9e3d59e30237788c/src/sql/run.py#L232
https://github.com/ploomber/jupysql/blob/8ce3334182733145dbea6ece9e3d59e30237788c/src/sql/run.py#L302

and tell people they should use the `%sqlplot` API

**Comment by tl3119** [comment]
So, for this issue, we should not use the `pie `and `bar` function in` run.py`. Instead, we should use `%sqlplot` defined in `plot.py`. So we need to show a warning in the `run.py` to tell the user they should use the `%sqlplot` API?

**Comment by edublancas** [comment]
correct.

**Comment by edublancas** [comment]
@bryannho let's work on getting the PR merged. you have some issues in the CI so merge from master. i think that'll fix it

**Comment by bryannho** [comment]
got it, will create a new PR to merge from master

**Comment by edublancas** [comment]
I see you created a new PR, that's fine but for next time there's no need to do it, you can just update the existing PR and merge from master to that PR
